drivers/misc/intel/comm_widget/intel_comm_widget_messages.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "intel_comm_widget.h"
+#include "pmc_interface.h"
```

Comment:
I think using relative paths is not a good thing.
Consider using \#include <zephyr/drivers/misc/intel_comm_widget/intel_comm_widget.h>
(just like other drivers do).
Or even better - move new headers to zephyr/include/zephyr/drivers/intel/comm_widget directory and include by
\#include <zephyr/drivers/intel/comm_widget/comm_widget.h>